test/util/marked-github-element.html

```diff
@@ -0,0 +1,157 @@
+<!doctype html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <title>marked-github-element tests</title>
+
+  <script src="../../src/bower_components/webcomponentsjs/webcomponents-lite.min.js"></script>
+  <script src="../../src/bower_components/web-component-tester/browser.js"></script>
+
+  <!-- Import the element to test -->
+  <link rel="import" href="../../src/util/marked-github-element.html">
+
+</head>
+<body>
+
+  <test-fixture id="basic">
+    <template>
+      <marked-github-element></marked-github-element>
+    </template>
+  </test-fixture>
+  <script>
+    suite('Basic <marked-github-element>', function() {
+      var mdElem;
+      var project = {
+        owner: 'SomeRepoOwner',
+        name: 'SomeRepoName'
+      }
+
+      setup(function() {
+        mdElem = fixture('basic');
+      });
+
+      test('Passes project owner and name to substitutor', function(done) {
+        mdElem.createMarkdownSubstituter = function(p) {
+          assert.deepEqual(p, project);
+          done();
+        }
+        mdElem.project = project;
+      });
+
+      test('passes markdown to substitutor', function(done) {
+        var markdown = 'testMD'
+        mdElem.createMarkdownSubstituter = function(p) {
+          console.log('create');
+          return function(err, text) {
+            assert.include(text, markdown);
```

Comment:
Because `text` is rendered markdown that includes HTML, and `markdown` is just a string